txgh/lib/txgh/events.rb

```diff
@@ -24,10 +24,16 @@ def channels
       channel_hash.keys
     end
 
-    def publish_error(e)
+    def publish_error(e, options = {})
       # if nobody has subscribed to error events, raise original error
-      callbacks = channel_hash.fetch(ERROR_CHANNEL) { raise e }
-      callbacks.each { |callback| callback.call(e) }
+      callbacks = channel_hash.fetch(ERROR_CHANNEL) do
+        raise(e) if options.fetch(:raise_if_no_subscribers, true)
```

Comment:
would it make sense to have this as a positively worded option?
`raise(e) if options.fetch(:subscribers, []).empty?`